src/applications/disability-benefits/all-claims/submit-transformer.js

```diff
@@ -354,6 +366,36 @@ export function transform(formConfig, form) {
     }
     return clonedData;
   };
+
+  const addForm8940 = formData => {
+    const clonedData = _.cloneDeep(formData);
+    const unemployability = clonedData.unemployability;
```

Comment:
you could do a pluck here `const { unemployability } = clonedData`. Little shorter 🤷‍♂️